zio/detector/file.go

```diff
@@ -40,14 +43,14 @@ func IsS3Path(path string) bool {
 // URL. If the path is neither of these or can't otherwise be opened,
 // an error is returned.
 func OpenFile(zctx *resolver.Context, path string, cfg OpenConfig) (*zbuf.File, error) {
-	if IsS3Path(path) {
-		return OpenS3File(zctx, path, cfg)
-	}
-
 	if cfg.Format == "parquet" {
```

Comment:
Consider adding a comment here. Something like:
```suggestion
        // Parquest is special and needs its own reader for s3 sources- therefore this must go before
        // the IsS3Path check.
	if cfg.Format == "parquet" {
```